cradle-cassandra/src/test/java/com/exactpro/cradle/cassandra/integration/messages/PageSessionsAndGroupsApiTest.java

```diff
@@ -0,0 +1,222 @@
+            b3.addMessage(generateMessage(SESSION_ALIAS6, Direction.SECOND, 28, 10L));
+
+            //page 4
+            GroupedMessageBatchToStore b4 = new GroupedMessageBatchToStore(GROUP4_NAME, 1024, storeActionRejectionThreshold);
+            b4.addMessage(generateMessage(SESSION_ALIAS6, Direction.FIRST, 35, 11L));
+            b4.addMessage(generateMessage(SESSION_ALIAS6, Direction.SECOND, 46, 12L));
+
+            List<GroupedMessageBatchToStore> data = List.of(b1, b2, b3, b4);
+
+            for (GroupedMessageBatchToStore el : data) {
+                storage.storeGroupedMessageBatch(el);
+            }
+
+        } catch (CradleStorageException | IOException e) {
+            logger.error(e.getMessage(), e);
+            throw new RuntimeException(e);
+        }
+    }
+
+    //------------------------------------------Session Alias test cases------------------------------------------------
+
+    @Test(description = "Simply gets all session aliases for a given book from database")
+    public void getAllSessionAliasesTest() throws CradleStorageException, IOException {
+        try {
+            var actual = storage.getSessionAliases(bookId);
+            Assertions.assertThat(actual.size()).isEqualTo(6);
+            Assertions.assertThat(actual).hasSameElementsAs(allSessions);
+        } catch (IOException | CradleStorageException e) {
+            logger.error(e.getMessage(), e);
+            throw e;
+        }
+    }
+
+    @Test(description = "Get session aliases fora a given book and a time interval that covers only one page")
+    public void getSessionAliasesOnePageNoDuplicatesTest() throws CradleStorageException {
+        try {
+            var resultSet = storage.getSessionAliases(bookId, new Interval(dataStart, dataStart.plus(9, ChronoUnit.MINUTES)));
+            var actual = newArrayList(resultSet.asIterable());
+            Assertions.assertThat(actual.size()).isEqualTo(2);
+            var expected = Set.of(SESSION_ALIAS1, SESSION_ALIAS2);
+            Assertions.assertThat(actual).hasSameElementsAs(expected);
+        } catch (CradleStorageException e) {
+            logger.error(e.getMessage(), e);
+            throw e;
+        }
+    }
+
+    @Test(description = "Get session aliases fora a given book and a time interval that covers several pages")
+    public void getSessionAliasesMultiPageNoDuplicatesTest() throws CradleStorageException {
+        try {
+            var resultSet = storage.getSessionAliases(bookId, new Interval(dataStart, dataStart.plus(13, ChronoUnit.MINUTES)));
+            var actual = newArrayList(resultSet.asIterable());
+            Assertions.assertThat(actual.size()).isEqualTo(4);
+            var expected = Set.of(SESSION_ALIAS1, SESSION_ALIAS2, SESSION_ALIAS3, SESSION_ALIAS4);
+            Assertions.assertThat(actual).hasSameElementsAs(expected);
+        } catch (CradleStorageException e) {
+            logger.error(e.getMessage(), e);
+            throw e;
+        }
```

Comment:
We can avoid catching because Unit test report should contains information about exception